python/pyenv/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    pyenv
+version                 2.3.2
+categories-append       python devel
```

Comment:
this probably fits better in  "sysutils" as the primary category?

Reply:
Cool - I picked devel on the grounds I've primarily used it as a dev tool but sysutils definitely makes sense in a more neutral context.